### DISCUSSION TITLE
FastAPI v0.100.0 is removing dict BaseModel attributes in response

### DISCUSSION THREAD

**Discussion by agabarre97** [fastapi/fastapi#9871] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class MyCustomModel(BaseModel):
    content: str
    created_at: datetime
    stats: dict
```

### Description

When the response reaches the validate function https://github.com/tiangolo/fastapi/blob/master/fastapi/routing.py#L144, the response content is a dict like: {"content": "asdasda", "created_at": asdasda, "stats": {}}

However, when this function applies the value returned contains the object but stats field. This function deletes this. This didn't happened in v0.83.0 but it's happening in v0.100.0

### Operating System

Linux

### FastAPI Version

0.100.0

### Python Version

3.8

**Comment by YuriiMotov** [comment]
Can't reproduce this. I think that was Pydantic issue.
```python
from fastapi import FastAPI, Response
from pydantic import BaseModel


app = FastAPI()


class MyCustomModel(BaseModel):
    content: str
    stats: dict


@app.get("/", response_model=MyCustomModel)
def get_status(id: str, response: Response):
    return {"content": "asdasda", "stats": {}}

```
Anyway, it works well in current version
